app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java

```diff
@@ -23,6 +27,14 @@
 @RunWith(AndroidJUnit4.class)
 public class GameLobbyActivityInstrumentedTest {
 
+    @BeforeClass
+    public static void setPersistence(){
```

Comment:
I think there is quite a lot of code duplication with this function, I think it is better to have a function where these 4 lines of code are executed and simply call that externally defined function from each of the `set Persistence() `function from each class.

Reply:
Since it is in the test I think it's fine.

Reply:
Oh, yes my bad sure of course!